[2024-04-16T18:04:22.780+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: files_ext.exst manual__2024-04-16T18:04:21.147172+00:00 [queued]>
[2024-04-16T18:04:22.790+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: files_ext.exst manual__2024-04-16T18:04:21.147172+00:00 [queued]>
[2024-04-16T18:04:22.791+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-16T18:04:22.803+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): exst> on 2024-04-16 18:04:21.147172+00:00
[2024-04-16T18:04:22.809+0000] {standard_task_runner.py:60} INFO - Started process 1347 to run task
[2024-04-16T18:04:22.811+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'files_ext', 'exst', 'manual__2024-04-16T18:04:21.147172+00:00', '--job-id', '2011', '--raw', '--subdir', 'DAGS_FOLDER/files_extraction.py', '--cfg-path', '/tmp/tmpec_w6mvp']
[2024-04-16T18:04:22.814+0000] {standard_task_runner.py:88} INFO - Job 2011: Subtask exst
[2024-04-16T18:04:22.851+0000] {task_command.py:423} INFO - Running <TaskInstance: files_ext.exst manual__2024-04-16T18:04:21.147172+00:00 [running]> on host fdd3cfdf3f53
[2024-04-16T18:04:22.914+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='files_ext' AIRFLOW_CTX_TASK_ID='exst' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T18:04:21.147172+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-16T18:04:21.147172+00:00'
[2024-04-16T18:04:22.996+0000] {logging_mixin.py:188} INFO - Error downloading file: Unable to locate credentials
[2024-04-16T18:04:22.996+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-16T18:04:23.004+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=files_ext, task_id=exst, execution_date=20240416T180421, start_date=20240416T180422, end_date=20240416T180423
[2024-04-16T18:04:23.024+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-16T18:04:23.036+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
